packages/suite/src/components/wallet/AccountsMenu/AccountItemSkeleton.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { SkeletonCircle, SkeletonRectangle, Stack } from '@suite-components/Skeleton';
+import { useLoadingSkeleton } from '@suite-hooks';
+import { AccountHeader, Left, Right } from './AccountItem';
+
+export const AccountItemSkeleton = (props: { animate?: boolean }) => {
```

Comment:
```suggestion
export const AccountItemSkeleton = ({ animate }: AccountItemSkeletonProps) => {
```

Reply:
it was just a draft so far, please check the second commit